webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -1948,7 +1948,15 @@ var MCK_BOT_MESSAGE_QUEUE = [];
                     var comment = document.getElementById("mck-feedback-comment");
                     comment && comment.value.trim() && (feedbackObject.comments = [comment.value]);
                     feedbackObject.rating = parseInt(document.querySelector('.mck-rating-box.selected').getAttribute("data-rating"));
-                    feedbackObject.groupId = CURRENT_GROUP_DATA.tabId;
+                    feedbackObject.groupId = CURRENT_GROUP_DATA && CURRENT_GROUP_DATA.tabId;
+                    feedbackObject.supportAgentName = CURRENT_GROUP_DATA && CURRENT_GROUP_DATA.conversationAssignee;
+                    feedbackObject.applicationId = MCK_APP_ID;
+                    let LOGGED_IN_USER = alUserService.MCK_USER_DETAIL_MAP[MCK_USER_ID];
```

Comment:
@Kusum-Dhoundiyal @ab14bhardwaj pls check if 'let' will work in chat plugin for all browsers?